parse/functions.go

```diff
@@ -300,6 +300,28 @@ var (
 				return fmt.Sprintf("array_length(%s, 1)", inputs[0]), nil
 			},
 		},
+		"array_remove": {
```

Comment:
Can we construct tests again the functions? At least test `PGFormat`

Reply:
We do, it is covered in an integration test. The integration test covers it. I felt this was most fitting because it tests that the generated postgres syntax is valid

Reply:
Can you show me where this test is? I couldn't locate it.